doc/source/theory/fluid_dynamics/vof.rst

```diff
@@ -0,0 +1,233 @@
+
+.. math::
+  \partial_t \phi + \partial_i \phi u_i = 0 \quad \forall (x_i,t)\in \Omega\times[0,T]
+
+Developing the second term gives:
+
+.. math::
+  \partial_t \phi + \phi\partial_i u_i + u_i\partial_i\phi = 0 \quad \forall (x_i,t)\in \Omega\times[0,T]
+
+Typically, the term :math:`\phi\partial_i u_i` (or :math:`\phi \nabla \cdot \vec{u}`) is zero due to mass conservation in the Navier-Stokes equations. However, previous work done in Lethe showed that while :math:`\nabla \cdot \vec{u}=0` is globally respected, it is not locally respected, especially around the interface, so lets keep it for now.
+
+To complete the strong formulation of the problem, let's impose a no flux boundary condition on :math:`\partial \Omega`:
+
+.. math::
+  \partial_i \phi n_i= 0 \quad \forall (x_i,t)\in \partial \Omega\times[0,T]
+
+where :math:`n_i` represent the outward pointing unit normal vector of :math:`\partial \Omega`, i.e., :math:`\vec{n}`.
+
+Finite element formulation
+---------------------------
+
+To obtain the finite element formulation, we first need the weak formulation. Therefore, let :math:`v` be an arbitrary continous scalar function on :math:`\Omega`. To obtained the weak form, we multiply the strong problem by :math:`v` and integrate over :math:`\Omega`:
+
+.. math::
+  \int_\Omega v \left( \partial_t \phi + \phi\partial_i u_i + u_i\partial_i\phi\right) d \Omega = 0
+
+To insure that those integrals are well defined in :math:`\Omega`, we chose the appropriate solution spaces:
+
+.. math::
+  \phi \in \Phi(\Omega) = H^1(\Omega)
+
+.. math::
+  v \in V(\Omega) = H^1(\Omega)
+
+Since :math:`\Phi=V`, the weak problem is:
+
+Find :math:`\phi \in V \times [0,T]` such that
+
+.. math::
+  \int_\Omega v \left( \partial_t \phi + \phi\partial_i u_i + u_i\partial_i\phi\right) d \Omega = 0 \quad \forall v\in V
+
+Using Galerkin method, the finite element formulation reads:
+
+Find :math:`\phi^h \in V^h \times [0,T]` such that
+
+.. math::
+  \int_\Omega v^h \left( \partial_t \phi^h + \phi^h\partial_i u_i + u_i\partial_i\phi^h\right) d \Omega = 0 \quad \forall v^h\in V^h\times [0,T]
+
+where :math:`V^h` is the finite element space, and :math:`\phi^h(\vec{x},t) = \sum_{j=1}^N \phi_j(t)\psi_j(\vec{x})`. In standard notation, this formulation corresponds to:
+
+Find :math:`\phi^h \in V^h \times [0,T]` such that
+
+.. math::
+  \int_\Omega v^h \left(\frac{\partial \phi^h}{\partial t} + \phi^h \nabla \cdot \vec{u} + \vec{u}\cdot \nabla \phi^h \right) d \Omega = 0 \quad \forall v^h\in V^h\times [0,T]
+
+Stabilization
+--------------
+
+The numerical resolution of the advection equation requires stabilization because of its purely convective character. Furthermore, a second stabilization term is added to improve the capturing of the interface due to sharp gradient across :math:`\Gamma` and perpendicular to the streamlines. To that end, a Discontinuity-Capturing Directional Dissipation (DCDD) shock capturing scheme is used [`2 <https://doi.org/10.1002/fld.505>`_]:
```

Comment:
```suggestion
The numerical resolution of the advection equation requires stabilization because of its purely advective character, which makes the equation hyperbolic. Furthermore, a second stabilization term is added to improve the capturing of the interface due to sharp gradient across :math:`\Gamma` . Since SUPG only adds diffusion along the streamlines, crosswind oscillation may occur if no appropriate shock capturing scheme are used. To that end, a Discontinuity-Capturing Directional Dissipation (DCDD) shock capturing scheme is used [`2 <https://doi.org/10.1002/fld.505>`_]:
```